bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/MagnitudeEditor.java

```diff
@@ -201,11 +202,11 @@ public void setTarget(SPTarget target, Mode mode) {
 
         @Override
         public Option<Magnitude.Band> getMagnitudeBand() {
-            return new Some<Magnitude.Band>(band);
+            return new Some<>(band);
         }
 
-        @Override public Option<JComboBox> getBandCombo() { return new Some<JComboBox>(cb); }
-        @Override public Option<JComboBox> getSystemCombo() { return new Some<JComboBox>(systemCb); }
+        @Override public Option<JComboBox> getBandCombo() { return new Some<>(cb); }
```

Comment:
`JComboBox` has been generalized in 1.7, I've been trying to make it generic when possible. Can it be done here?